enterprise/reporting/error_extractor.go

```diff
@@ -31,21 +32,52 @@ var (
 	whitespacesRegex = regexp.MustCompile("[ \t\n\r]*") // used in checking if string is a valid json to remove extra-spaces
 
 	defaultErrorMessageKeys = []string{"message", "description", "detail", "title", errorKey, "error_message"}
+	deprecationKeywords     = map[string]int{
```

Comment:
Can we ensure lowercase of keys somehow (to make sure we don't have casing issues for the keys added in the future)?

Reply:
Are you asking to deliberately lowercase even though this map now contains lowercased keys ?

Reply:
Done